base_import_module_group/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020-2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Import Module Group",
+    "summary": "Restrict module importation to specific group",
+    "version": "13.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto-install": True,
```

Comment:
There is a typo here, see [this PR](https://github.com/OCA/server-tools/pull/2367)